tests/dataDriven_test.js

```diff
@@ -0,0 +1,27 @@
+let data = new DataTable(["url", "statusCode"]);
+data.add([
+  "/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=196167379",
+  "200",
+]);
+data.add([
+  "/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=098734232",
+  "404",
+]);
+
+var assert = require("assert");
+Feature("Data Driven");
+
+Scenario("I can verify the URL returns a status code of 404", async ({ I }) => {
+  let res = await I.sendGetRequest(
```

Comment:
Remember to add a // Given comment here stating where the Scenario starts from

Reply:
Comments added to the test file

Reply:
The `// When` comment is unnecessary here as it says the same thing as the `// Given` - it's OK to not always have a `// When`